Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -69,6 +69,26 @@ final class PackageToolTests: XCTestCase {
             XCTAssertEqual(GitRepository(path: path).tags, ["1.2.3", "1.2.4"])
         }
     }
+    
+    func testUpdateDryRun() throws {
```

Comment:
It would be better if you can write a unit test since they're much much faster to run. You can see `WorkspaceTests.swift` for examples. Don't worry about it if it's turning out to be too hard.